src/ATen/test/CMakeLists.txt

```diff
@@ -5,10 +5,13 @@ ADD_EXECUTABLE(basic basic.cpp)
 TARGET_LINK_LIBRARIES(basic ATen)
 
 add_executable(atest atest.cpp)
-target_link_libraries(atest ATen)
+TARGET_LINK_LIBRARIES(atest ATen)
```

Comment:
use `target_link_libraries`

Reply:
there is inconsistency about this in the codebase, should I make it consistent throughout?

Reply:
Just keep the diff small :)